tox.ini

```diff
@@ -7,9 +7,11 @@ whitelist_externals = make
 usedevelop = True
 extras =
     test
+passenv = PGPASSWORD PGHOST PGUSER PGPORT
 commands =
     /usr/bin/find . -name '*.pyc' -delete
     pip freeze -l
+    pytest --cov-report term-missing --cov=west tests/unittests
```

Comment:
so we don't launch the integration tests in the CI?

Reply:
coming in the next PR :)